nodejs/fastify/dns_api/dns.sql

```diff
@@ -0,0 +1,17 @@
+﻿--desc: basic
+SELECT
+    SYSTOOLS.HTTPGETCLOB(
+        URL => CAST('http://localhost:8088/dns/' CONCAT 'yahoo.com' AS VARCHAR(255)),
+        HTTPHEADER => CAST(NULL AS CLOB(1K))) AS ADDR
+    FROM SYSIBM.SYSDUMMY1;    
```

Comment:
```suggestion
VALUES(SYSTOOLS.HTTPGETCLOB(
        URL => CAST('http://localhost:8088/dns/' CONCAT 'yahoo.com' AS VARCHAR(255)),
        HTTPHEADER => CAST(NULL AS CLOB(1K))) AS ADDR);
```

Using VALUES() is cleaner

Reply:
I tried running this and got `[SQL0104] Token SYSTOOLS was not valid.`. Any ideas?

Reply:
Perhaps it should be `values SYSTOOLS.HTTPGETCLOB(URL => cast('http://localhost:8088/dns/' concat 'yahoo.com' as varchar(255)), HTTPHEADER => cast(NULL as clob(1k)));`